java-generator/cli/pom.xml

```diff
@@ -62,6 +62,39 @@
       <version>${slf4j.version}</version>
     </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>docker</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>jib-maven-plugin</artifactId>
+            <version>3.2.0</version>
+            <configuration>
+              <to>
+                <image>fabric8io/java-generator-cli</image>
+                <tags>
+                  <tag>${project.version}</tag>
+                  <tag>latest</tag>
+                </tags>
+              </to>
+            </configuration>
+            <executions>
+              <execution>
+                <phase>package</phase>
+                <goals>
+                  <goal>dockerBuild</goal>
```

Comment:
This builds to a local Docker daemon. Reviewer to verify if there is a public container registry this can be published to.